CREATE_WORKLOAD_GUIDE.md

```diff
@@ -15,32 +15,87 @@ $ opensearch-benchmark create-workload \
 --target-hosts="<CLUSTER ENDPOINT>" \
 --client-options="basic_auth_user:'<USERNAME>',basic_auth_password:'<PASSWORD>'" \
 --indices="<INDICES TO GENERATE WORKLOAD FROM>" \
+--number-of-docs="<INDEX NAME>:<NUMBER OF DOCUMENTS>" 
 --output-path="<LOCAL DIRECTORY PATH TO STORE WORKLOAD>"
+--custom-queries="<FILE PATH OF CUSTOM QUERIES TO ADD FOR THE WORKLOAD EXECUTION>"
```

Comment:
Could you rephrase this? Something like "file path of custom queries to add to the workload" would be more clear